syntax = "proto3";

package moose;

service Executor {
    rpc RunComputation(RunComputationRequest) returns(RunComputationResponse) {}
}

message RunComputationRequest {
    bytes computation = 1;
    string placement = 2;
    int64 session_id = 3;
}

message RunComputationResponse {}
